src/components/Heading/index.js

```diff
@@ -1,26 +1,66 @@
+import { css } from '@emotion/core';
 import invariant from 'invariant';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-export const Heading = ({ children, level, ...otherProps }) => {
+import { useTheme } from 'themes';
+import { headingLarge, headingMedium, headingSmall } from 'themes/mixins';
+
+export const Heading = ({
+  children,
+  level,
+  large,
+  medium,
+  small,
+  ...otherProps
+}) => {
   invariant(level, 'Heading component requires a level prop');
 
   const HeadingElement = `h${level}`;
+  const theme = useTheme();
+
+  return (
+    <HeadingElement
+      css={css`
+        ${css(headingMedium(theme))};
+        margin: 0 0 ${theme.spacing.margin.m};
+
+        ${large &&
+          css`
+            ${css(headingLarge(theme))}
+          `}
 
-  return <HeadingElement {...otherProps}>{children}</HeadingElement>;
+        ${small &&
+          css`
+            ${css(headingSmall(theme))}
+          `}
+      `}
+      {...otherProps}
+    >
+      {children}
+    </HeadingElement>
+  );
 };
 
 export const HeadingLevels = [1, 2, 3, 4, 5, 6];
 
 Heading.defaultProps = {
   children: undefined,
+  level: 2,
+  large: true,
```

Comment:
I'll fix this for you, but you don't want to set boolean props to `true` if you accept other conflicting boolean props. If I make it so mixing large/medium would cause an error, `<Heading medium>` would throw an error because `large` is implicitly set to `true`. You'd need to do `<Heading large={false} medium>` to make it work. We'll set `large` based on the supplied level.